README.md

```diff
@@ -17,7 +17,9 @@ features for JSON Pointers as well.
 
 ## Documentation
 
-The documentation for this project can be found here: https://github.com/whitlockjc/json-refs/blob/master/docs/README.md
+The documentation for this project can be found here: https://github.com/whitlockjc/json-refs/blob/master/docs/README.md .
+
+The API can be found at https://github.com/whitlockjc/json-refs/blob/master/docs/API.md .
```

Comment:
Minor consistency difference as well.   The first uses `{some message}: {some url}` where the second uses `{some message} {some url}`.  The colon is missing.  Minor nit but if we're fixing it, we might as well be consistent.

Update the PR with your pick of the two approaches and I'll get it merged.